### ISSUE TITLE
netbox: Forbidden (403) after login

### ISSUE THREAD

**Issue by benek984** [casperklein/homeassistant-addons#12]
Hello,
Installed addon as instructed. Started add-on going to https://192.168.x.x:5580 trying to login with credentials specified in configuration getting this:

<img width="802" alt="image" src="https://user-images.githubusercontent.com/62546477/235761629-1ffa0d3b-5afa-4b0e-af67-5918cea7defd.png">

Also trying to go to add-on web UI by clicking Open Web UI doesn't work.

**Comment by casperklein** [comment]
I cannot reproduce your issue. Which version are you using? Did you check the addon log?

**Comment by benek984** [comment]
````
Starting redis-server: redis-server.
Applying database migrations..
Operations to perform:
  Apply all migrations: admin, auth, circuits, contenttypes, core, dcim, django_rq, extras, ipam, sessions, social_django, taggit, tenancy, users, virtualization, wireless
Running migrations:
  No migrations to apply.
Collecting static files..
283 static files copied to '/opt/netbox/netbox/static', 229 unmodified.
Starting TLS tunnels: /etc/stunnel/stunnel.conf: started
[03/May/2023 14:33:19] Housekeeping..
[*] Clearing expired authentication sessions
	Sessions cleared.
[*] Checking for expired changelog records
	No expired records found.
[*] Checking for expired jobs
	No expired records found.
[*] Checking for latest release
	Skipping: RELEASE_CHECK_URL not set
Finished.
Starting RQ worker process..
Starting netbox..
Performing system checks...
13:33:30 Worker rq:worker:cbba8b86472148e8a6f520e3c6fcb08d: started, version 1.13.0
13:33:30 Subscribing to channel rq:pubsub:cbba8b86472148e8a6f520e3c6fcb08d
13:33:30 *** Listening on high, default, low...
13:33:30 Cleaning registries for queue: high
13:33:30 Cleaning registries for queue: default
13:33:30 Scheduler for low,default,high started with PID 88
13:33:30 Cleaning registries for queue: low
System check identified no issues (0 silenced).
May 03, 2023 - 13:33:32
Django version 4.1.8, using settings 'netbox.settings'
Starting development server at http://0.0.0.0:443/
Quit the server with CONTROL-C.
[03/May/2023 13:34:07] "GET / HTTP/1.1" 200 27521
[03/May/2023 13:34:08] "GET /static/netbox-external.css?v=3.5.0 HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/setmode.js HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/netbox_icon.svg HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/netbox-print.css?v=3.5.0 HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/netbox.js?v=3.5.0 HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/netbox-dark.css?v=3.5.0 HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/netbox-light.css?v=3.5.0 HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/netbox_logo.svg HTTP/1.1" 304 0
[03/May/2023 13:34:08] "GET /static/materialdesignicons-webfont-ER2MFQKM.woff2?v=7.0.96 HTTP/1.1" 304 0
[03/May/2023 13:34:12] "GET /login/?next=/ HTTP/1.1" 200 4926
[03/May/2023 13:34:14] "POST /login/ HTTP/1.1" 403 1019

````

**Comment by casperklein** [comment]
Can you check, if it works with HTTP instead of HTTPS?

**Comment by casperklein** [comment]
The issue should be fixed with the latest addon version (3.5.1). Please check if that works for you.

**Comment by benek984** [comment]
All good now. Thank you
